### PR TITLE
Add support for Fedora 30

### PR DESCRIPTION
**Pull request by mikedep333**
Requires https://github.com/geerlingguy/ansible-role-postgresql/pull/53 first.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -12,6 +12,11 @@ galaxy_info:
       versions:
         - 6
         - 7
+    - name: Fedora
+      versions:
+        - 27
+        - 28
+        - 29
```
Comment (geerlingguy) [inline meta/main.yml]: This would also need `30` and maybe we can take off 27/28 since they are EOL'ed?

File: tasks/variables.yml [hunk 2]
```diff
@@ -6,7 +6,11 @@
 
 - name: Include OS-specific variables (RedHat).
   include_vars: "{{ ansible_os_family }}-{{ ansible_distribution_version.split('.')[0] }}.yml"
-  when: ansible_os_family == 'RedHat'
+  when: ansible_os_family == 'RedHat' and ansible_distribution != 'Fedora'
```
Comment (geerlingguy) [inline tasks/variables.yml]: Can you format this as:

```yaml
when:
  - ansible_os_family == 'RedHat'
  - ansible_distribution != 'Fedora'
```

Just makes it easier to read (IMO) and is the pattern I use elsewhere for ANDs

File: vars/Fedora-27.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+__postgresql_version: "9.6"
+__postgresql_data_dir: "/var/lib/pgsql/data"
+__postgresql_bin_path: "/usr/bin"
+__postgresql_config_path: "/var/lib/pgsql/data"
+__postgresql_daemon: postgresql
+__postgresql_packages:
+  - postgresql
```
Comment (geerlingguy) [inline vars/Fedora-27.yml]: This file can be removed since 27 is EOL

File: vars/Fedora-28.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+__postgresql_version: "10.5"
+__postgresql_data_dir: "/var/lib/pgsql/data"
+__postgresql_bin_path: "/usr/bin"
+__postgresql_config_path: "/var/lib/pgsql/data"
+__postgresql_daemon: postgresql
+__postgresql_packages:
+  - postgresql
```
Comment (geerlingguy) [inline vars/Fedora-28.yml]: This file can be removed since 28 is EOL

**Comment by mikedep333** [comment]
> Please see review comments—additionally, if you could modify .travis.yml and add `fedora30` to the test suite, that would be helpful in making sure the changes work and continue to work on Fedora 30 :)

Done the best I could. It looks like you only have an image for Fedora 29.

File: tasks/variables.yml (hunk 2 above)
Comment (mikedep333) [reply tasks/variables.yml]: Done; I agree with this style.

File: vars/Fedora-27.yml (hunk 3 above)
Comment (mikedep333) [reply vars/Fedora-27.yml]: Done.

File: vars/Fedora-28.yml (hunk 4 above)
Comment (mikedep333) [reply vars/Fedora-28.yml]: Done

File: meta/main.yml (hunk 1 above)
Comment (mikedep333) [reply meta/main.yml]: Done

**Comment by mikedep333** [comment]
> Please see review comments—additionally, if you could modify .travis.yml and add `fedora30` to the test suite, that would be helpful in making sure the changes work and continue to work on Fedora 30 :)

I added this after all. Sorry for the earlier confusion.

**Comment by mikedep333** [comment]
@geerlingguy I worked on this about a month ago, but I believe I implemented all of your requested changes completely.

**Comment by lhoss** [comment]
All requested changes implemented and build is *green* , let's merge 👍 
After that, I can rebase those changes onto https://github.com/geerlingguy/ansible-role-postgresql/pull/54, which I'm particularly interested in

**Pull request merged**